docs/Build-for-NuttX-(new).md

```diff
@@ -1,40 +1,40 @@
 #### About
 
-This page describes short version to use with latest version of NuttX for STM32F4-Discovery with BB.
+This page describes a short version to use with the latest version of NuttX for STM32F4-Discovery with BB.
 
-Please read [Build for NuttX](https://github.com/Samsung/iotjs/wiki/Build-for-NuttX) page and understand full procedure.
+Please read the [Build for NuttX](https://github.com/Samsung/iotjs/wiki/Build-for-NuttX) page and understand full procedures.
 
 #### Get NuttX for IoT.js
 
-Follow [Building nuttx for iotjs](https://bitbucket.org/seanshpark/nuttx/wiki/Home) page.
+Follow the [Building nuttx for iotjs](https://bitbucket.org/seanshpark/nuttx/wiki/Home) page.
 
 You should first build NuttX before building IoT.js, it depends on system header files. When NuttX tells you that it needs `libtuv` then come here again and continue with below sections.
 
 #### Build IoT.js for NuttX
 
-Giving these options are needed
+These options has to be needed.
```

Comment:
I'm not sure this sentence is grammatically correct. Previous version was better :)

> These options are needed